### ISSUE TITLE
wrong type for load_factor in ServerInfo

### ISSUE THREAD

**Issue by tommq** [XRPLF/xrpl4j#85]
I started getting Exceptions from serverInfo() method when connected to s2.ripple.com. I realized the library expects load_factor  to be UnsignedInteger (and it usually is "1"), but rippled sometimes returns it as double.

It can be shown here: https://xrpl.org/websocket-api-tool.html?server=wss%3A%2F%2Fs1.ripple.com%2F&req=%7B%22id%22%3A1%2C%22command%22%3A%22server_info%22%7D

clicking send several times eventually get you response with some higher load like:

```
{
  "id": 1,
  "result": {
    "info": {
      "build_version": "1.7.0",
      "complete_ledgers": "61881385-62562429",
      "hostid": "LARD",
      "io_latency_ms": 2,
      "jq_trans_overflow": "0",
      "last_close": {
        "converge_time_s": 3.002,
        "proposers": 38
      },
      "load_factor": 511.83203125,
      "load_factor_server": 1,
      "peer_disconnects": "224008",
      "peer_disconnects_resources": "747",
      "peers": 261,
      "pubkey_node": "n9MozjnGB3tpULewtTsVtuudg5JqYFyV3QFdAtVLzJaxHcBaxuXD",
      "server_state": "full",
      "server_state_duration_us": "2274468435925",
      "state_accounting": {
        "connected": {
          "duration_us": "231197490",
          "transitions": 2
        },
        "disconnected": {
          "duration_us": "1156933",
          "transitions": 2
        },
        "full": {
          "duration_us": "2274468435925",
          "transitions": 1
        },
        "syncing": {
          "duration_us": "4065001",
          "transitions": 1
        },
        "tracking": {
          "duration_us": "63",
          "transitions": 1
        }
      },
      "time": "2021-Mar-30 15:37:51.486384 UTC",
      "uptime": 2274704,
      "validated_ledger": {
        "age": 4,
        "base_fee_xrp": 0.00001,
        "hash": "E5A958048D98D4EFEEDD2BC3F36D23893BBC1D9354CB3E739068D2DFDE3D1AA3",
        "reserve_base_xrp": 20,
        "reserve_inc_xrp": 5,
        "seq": 62562429
      },
      "validation_quorum": 31
    }
  },
  "status": "success",
  "type": "response"
}
``` 

The exception I am getting is clearly saying that jackson can't construct UnsignedInteger from 512.90234375
```
com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `com.google.common.primitives.UnsignedInteger` (although at least one Creator exists): no double/Double-argument constructor/factory method to deserialize from Number value (512.90234375)
 at [Source: (String)"{"info":{"build_version":"1.7.0","complete_ledgers":"32570-62562188","hostid":"CARR","io_latency_ms":1,"jq_trans_overflow":"2942","last_close":{"converge_time_s":3.005,"proposers":38},"load_factor":512.90234375,"load_factor_server":1,"peer_disconnects":"171040","peer_disconnects_resources":"4176","peers":214,"pubkey_node":"n9Kr6UDJAd2yUn9J6bXWpRvQcKRPAEVpeV33NJVBcaDSDVt2eho2","server_state":"full","server_state_duration_us":"172918538055","state_accounting":{"connected":{"duration_us":"471959630"[truncated 524 chars]; line: 1, column: 199] (through reference chain: org.xrpl.xrpl4j.model.client.server.ImmutableServerInfoResult$Json["info"]->org.xrpl.xrpl4j.model.client.server.ImmutableServerInfo$Json["load_factor"])
```

I suppose there fields like load_factor_local, load_factor_net and others may be the same case. I will try to investigate this to find the proper type later.

**Comment by nkramer44** [comment]
@tommq Nice catch.  I think this is either a bug in the documentation, or I just didn't read them correctly. I think changing this to a `Double` is probably sufficient, along with changing the other `load` fields to `Double`.

An even better solution here would be to create a wrapped `Load` type and type all load fields as such. We can just add another abstract class to `Wrappers`, `_Load extends Wrapper<Double>`.

If you're up for submitting a PR it would be much appreciated :)

**Comment by tommq** [comment]
Ok, I will submit a PR.

I also noticed that _jqTransOverflow, loadFactor,_ peers should be Optionals. I got the following error:

```
Cannot build ServerInfo, some of required attributes are not set
        [jqTransOverflow, loadFactor, peers]
```

**Comment by nkramer44** [comment]
Fixed in #87
